src/components/StudyPane/InfoPane/Motif/CategoryBlock.tsx

```diff
@@ -0,0 +1,52 @@
+import { useContext } from "react";
+import { DEFAULT_BORDER_COLOR, DEFAULT_COLOR_FILL, DEFAULT_TEXT_COLOR, FormatContext } from "../..";
+
+export const CategoryBlock = ({
+    category,
+    index,
+    selectedCategory,
+    setSelectedCategory,
+    value
+}: {
+    category: String,
+    index: number,
+    selectedCategory: String,
+    setSelectedCategory: React.Dispatch<React.SetStateAction<String>>,
+    value: { strongNumbers: number[], count: number }
+}) => {
+    const { ctxSetCategoryRoots } = useContext(FormatContext);
+
+    const handleClick = () => {
+        if(category == selectedCategory){
+            setSelectedCategory("");
+            ctxSetCategoryRoots([]);
+        }else{
+            setSelectedCategory(category);
+            ctxSetCategoryRoots(value.strongNumbers);
```

Comment:
Can we just use ctxSetSelectedHebWords? Root block also uses ctxSetSelectedHebWords